tango/common/util.py

```diff
@@ -186,6 +186,11 @@ def filename_is_safe(filename: str) -> bool:
     return all(c in SAFE_FILENAME_CHARS for c in filename)
 
 
+def make_safe_filename(name: str) -> str:
+    name = name.replace(" ", "-")
+    return "".join(c for c in name if c in SAFE_FILENAME_CHARS)
```

Comment:
This can map multiple different `name`s to the same filename. Maybe use urlencode instead?

Reply:
I don't think we'd want to encode this like a URL. You'd get URL escape codes in there which aren't file safe names. But we could append a hash of the original name to the resulting filename?

Reply:
Also with the way we're using this it would be very unlikely to run into name clashes. So far we only use it to map W&B or Beaker workspace names, which themselves have character restrictions, to directories.

Reply:
I'm OK with appending a hash. I realize collisions are unlikely, but time-consuming to debug when they happen. So I'd rather be safe.

Reply:
Done: 49a965a